encoder/libav_encoder.cpp

```diff
@@ -173,17 +173,29 @@ void LibAvEncoder::initVideoCodec(VideoOptions const *options, StreamInfo const
 	// Apply general options.
 	encoderOptionsGeneral(options, codec_ctx_[Video]);
 
-	const char *format;
-	if (options_->output.empty() && options->libav_format.empty())
-		format = "h264";
-	else if (options->libav_format.empty())
-		format = nullptr;
+	const std::string tcp { "tcp://" };
+	const std::string udp { "udp://" };
```

Comment:
I'm assuming we don't care about case here!

Reply:
I think this is case sensitive in libav.